src/pages/grade.tsx

```diff
@@ -13,11 +13,7 @@ import { TypeFormSubmission } from "../server/router/reviewers";
 const GradingPortal: NextPage = () => {
   const [togglePriotity, setTogglePriority] = useState(true);
 
-  const { data, isLoading } = trpc.useQuery([
-    togglePriotity
-      ? "reviewer.getPriorityApplications"
-      : "reviewer.getApplications",
-  ]);
+  const {data, isLoading} = togglePriotity ? trpc.reviewer.getPriorityApplications.useQuery() : trpc.reviewer.getApplications.useQuery();
```

Comment:
I wouldn't do this, it breaks the hook rules. I had this fixed by using the enabled option

Reply:
ok I'll change that thanks